docs/writing-docs/autodocs.md

```diff
@@ -111,6 +111,69 @@ Then you can use it in your `.storybook/preview.js` or an individual story file
 
 </div>
 
+### Generate a table of contents
+
+Storybook's auto-generated documentation pages can be quite long and difficult to navigate. To help with this, you can enable the table of contents feature to provide a quick overview of the documentation page and allow users to jump to a specific section. To enable it, extend your Storybook UI configuration file (i.e., `.storybook/preview.js`) and provide a `docs` [parameter](../writing-stories/parameters.md#global-parameters) with a `toc` property.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-preview-enable-toc.js.mdx',
+    'common/storybook-preview-enable-toc.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+### Configure the table of contents
+
+To simplify navigation, the table of contents on the documentation page will only show the `h3` headings that are automatically generated. However, if you want to customize the table of contents, you can add more parameters to the `toc` property. The following options and examples of how to use them are available.
```

Comment:
```suggestion
By default, the table of contents on the documentation page will only show the `h3` headings that are automatically generated. However, if you want to customize the table of contents, you can add more parameters to the `toc` property. The following options and examples of how to use them are available.
```

Reply:
Appreciate the suggestion, fixed with the latest commit